src/main/scala/com.snowplowanalytics.dataflow/streaming/storage/BigtableUtils.scala

```diff
@@ -0,0 +1,116 @@
+ * "AggregateRecords" table.
+ */
+object BigtableUtils {
+
+  val dateFormatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
+  val timezone = TimeZone.getTimeZone("UTC")
+
+  /**
+   * Function timezone helper
+   */
+  def timeNow(): String = {
+    dateFormatter.setTimeZone(timezone)
+    dateFormatter.format(new Date())
+  }
+
+
+  /**
+   * Function wraps get or create item in Bigtable table. Assuming single column family, called 'cf1'
+   */
+  def setOrUpdateCount(bigtableConnection: Connection, tableName: String, bucketStart: String, eventType: String, createdAt: String,  updatedAt: String, count: Int){
+
+    val rowInTable = getItem(bigtableConnection, tableName, bucketStart, eventType)
+    println("row in table:" + rowInTable)
+    if (rowInTable.isEmpty) {
+      BigtableUtils.putItem(bigtableConnection, tableName, bucketStart, eventType, createdAt, updatedAt, count)
+    } else {
+      val oldCreatedAt = new String(rowInTable.getValue("cf1".getBytes, "CreatedAt".getBytes))
+      val oldCount = new String(rowInTable.getValue("cf1".getBytes, "Count".getBytes)).toInt
+      val newCount = oldCount + count.toInt
+      BigtableUtils.putItem(bigtableConnection, tableName, bucketStart, eventType, oldCreatedAt, updatedAt, newCount)
+    }
+  }
+
+
+  /**
+   * Function wraps get row operation from Bigtable
+   */
+  def getItem(bigtableConnection: Connection, tableName: String, bucketStart: String, eventType: String): Result = {
+
+    val table = bigtableConnection.getTable(TableName.valueOf(tableName))
+    val getResult = table.get(new Get((bucketStart+":"+eventType).getBytes))
+    getResult
+  }
+
+
+  /**
+   * Function wraps Bigtable putItem operation
+   */
+  def putItem(bigtableConnection: Connection, tableName: String, bucketStart: String, eventType: String, createdAt: String,  updatedAt: String, count: Int) {
+
+    // Row column names
+    val tableEventTypeSecondaryKeyName = "EventType".getBytes
+    val tableCreatedAtColumnName = "CreatedAt".getBytes
+    val tableUpdatedAtColumnName = "UpdatedAt".getBytes
+    val tableCountColumnName = "Count".getBytes
+
+    try {
+      val table = bigtableConnection.getTable(TableName.valueOf(tableName))
+      println("Adding data to " + tableName)
+
+
+      // key composed of "bucketStart:eventType"
+      val put = new Put((bucketStart+":"+eventType).getBytes)
+        .addColumn("cf1".getBytes, tableEventTypeSecondaryKeyName, eventType.getBytes)
+        .addColumn("cf1".getBytes, tableCreatedAtColumnName, createdAt.getBytes)
+        .addColumn("cf1".getBytes, tableUpdatedAtColumnName, updatedAt.getBytes)
+        .addColumn("cf1".getBytes, tableCountColumnName, (""+count).getBytes)
+
+      // saving the data to Bigtable
+      // println(item)
+      table.put(put)
+    } catch {
+      case e: Exception => {
+        System.err.println("Failed to create item in " + tableName)
+        System.err.println(e.getMessage)
+      }
+    }
+  }
```

Comment:
Same, avoid println, lots of hardcoded strings and try to handle exceptions better